client/src/components/AnswerCard.js

```diff
@@ -3,13 +3,13 @@ import styled from '@emotion/styled';
 
 const AnswerContainer = styled.button`
   display: flex;
```

Comment:
I think you can center the content of `<button>` by using `margin: 0 auto` on its children. You should consider checking all your elements for `display: flex` and see if you could get child elements centred via `margin`

Reply:
Thanks, will do next round refactoring!